plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -445,6 +448,38 @@ function executeCLI($scriptName, &$args) {
 		$this->usage($scriptName);
 	}
 
+	/**
+	 * Pull out getopt style long options.
```

Comment:
What about actually using PHP's `getopt` function?

Reply:
I didn't think it could work without changing the existing CLI api. It seems like getopt only works if all the arguments conform to the getopt syntax.

Reply:
Gotcha, just a thought. It might be nice to move this toolset up to the parent class but that can wait for another use case. :+1: